### PR TITLE
Revising sleep window.

### PR DESCRIPTION
**Pull request by Nugs**
Add wake-adjustment - configure period for waking up early ahead of snipe time 
Add listing of distinct booking types available to help with future snipes 
Removal of thread from log entries (not really required)

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -247,46 +239,33 @@ venues {
...
     dining-types = ["Indoor", "Piazza"]
   }
-
   "Misi" {
     id = "2015"
     hour-of-day-to-start-booking = 10
     advance = "28 days"
     dining-types = ["Dining Room"]
   }
-
```
Comment (Nugs) [inline src/main/resources/application.conf:282]: Removed duplicate

File: src/main/resources/logback.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     <appender name="STDOUT" class="ConsoleAppender">
         <encoder class="PatternLayoutEncoder">
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} -%kvp- %msg%n</pattern>
```
Comment (Nugs) [inline src/main/resources/logback.xml:8]: No real need for thread logging at the moment - bit cleaner in the logs.

File: src/main/scala/com/resy/BookReservationWorkflow.scala [hunk 3]
```diff
@@ -95,9 +95,14 @@ object BookReservationWorkflow extends StrictLogging {
         (v \ "config" \ "type").asOpt[String],
         (v \ "config" \ "token").asOpt[String]
       )}
+      .tap(listBookingTypes)
```
Comment (Nugs) [inline src/main/scala/com/resy/BookReservationWorkflow.scala:98]: Grab the list of dining-types available; not a comprehensive list as it's only those available for booking, but will help with future accuracy of sniping hopefully.

File: src/main/scala/com/resy/BookingDetails.scala [hunk 4]
```diff
@@ -37,6 +38,6 @@ final case class BookingDetails(
 
   def inBookingWindow(leadTime: Int): Boolean = bookingWindowStart(leadTime).isBefore(LocalDateTime.now())
 
-  def minutesToBookingWindowStart(leadTime: Int): FiniteDuration =
-    (bookingWindowStart(leadTime).toEpochSecond(ZoneOffset.UTC) - LocalDateTime.now().toEpochSecond(ZoneOffset.UTC)).seconds.toMinutes.minutes
+  def secondsToBookingWindowStart(leadTime: Int): FiniteDuration =
```
Comment (Nugs) [inline src/main/scala/com/resy/BookingDetails.scala:41]: Switch to second precision to allow more precise start timing.

File: src/main/scala/com/resy/ResyBookingBot.scala [hunk 5]
```diff
@@ -74,9 +74,9 @@ object ResyBookingBot extends App with StrictLogging {
     logger.info("Within booking window for venue - attempting immediate reservation")
     bookReservationWorkflow()
   } else {
-    val durationToSleep: FiniteDuration = config.minutesToBookingWindowStart(leadTime)
+    val durationToSleep: FiniteDuration = config.secondsToBookingWindowStart(leadTime) - config.wakeAdjustment
```
Comment (Nugs) [inline src/main/scala/com/resy/ResyBookingBot.scala:77]: Add a wakeAdjustment to allow the bot to start sniping a few seconds prior to the booking window opening to allow for clcok inconsistencies.

File: src/main/resources/application.conf [hunk 6]
```diff
@@ -108,7 +100,7 @@ venues {
     id = "54591"
     hour-of-day-to-start-booking = 10
     advance = "13 days"
-    dining-types = ["Dining"]
+    dining-types = ["INDOOR", "OUTDOOR"]
```
Comment (Nugs) [inline src/main/resources/application.conf:103]: Aligning with values returned in latest check

File: src/main/resources/application.conf [hunk 7]
```diff
@@ -91,7 +83,7 @@ venues {
     dining-types = ["Outdoor", "Counter", "High Top w Backless Stools", "Booth"]
     info = "Closed Sat and Sunday/Walk ins Only on Monday - Largest Table is a Table of 8"
   }
-  "Una Pizza Napoletna" {
+  "Una Pizza Napoletana" {
```
Comment (Nugs) [inline src/main/resources/application.conf:86]: Typo correction

File: src/main/resources/application.conf [hunk 8]
```diff
@@ -54,10 +49,7 @@ venues {
     name = "L'Artusi"
     hour-of-day-to-start-booking = 8
     advance = "14 days"
-    dining-types = [
-      "Indoor Dining",
-      "Outdoor Patio"
-    ]
+    dining-types = ["Bar Counter", "Indoor Dining", "Patio Dining", "Counter"]
```
Comment (Nugs) [inline src/main/resources/application.conf:52]: Types retrieved from API

File: .gitignore [hunk 9]
```diff
@@ -20,7 +20,4 @@ project/target/
 .DS_Store
 .idea
 .scala_dependencies
-.worksheet
```
Comment (Nugs) [inline .gitignore:23]: No longer need this exclusion

**Comment by bfiegoli** [comment]
All changes look amazing!

**Pull request merged**